std-bits/table/src/main/java/org/enso/table/data/column/storage/numeric/DoubleStorage.java

```diff
@@ -289,6 +302,13 @@ protected long doOperation(double a) {
                 return (long) Math.floor(a);
               }
             })
+        .add(
+            new DoubleLongBooleanOpWithSpecialNumericHandling(Maps.ROUND) {
+              @Override
+              protected double doLongBoolean(double n, long decimalPlaces, boolean useBankers, int ix, MapOperationProblemBuilder problemBuilder) {
+                return Core_Math_Utils.roundDouble(n, decimalPlaces, useBankers);
+              }
```

Comment:
hat is the expected value type of:
```
t = Table.new [["X", [1.0, 2.3, 1123.123]]]
x = t.at "X"
x.value_type == Value_Type.Float

r = x.round decimal_places=0
r.value_type == ???
```

I assume it should be `Value_Type.Integer`, right?

We will need special logic to construct a different column type based on `decimal_places`